README.md

```diff
@@ -13,7 +21,7 @@ funcpack pack ./
 
 You can then test locally using the CLI tool: `func run <myfunc>`
 
-When uploading your files, you need to include your `package.json`, but you don't need your `node_modules` directory.
+When uploading your files, you need to include the single `.funcpack` directory (in the Functions App root), but you don't need your `node_modules` directory for each Function.
```

Comment:
You shouldn't include a node_modules folder for each Functions. Ideally, you have one set of dependencies. That helps reduce the size of your final pack.